Source/JSONAPISerializer.swift

```diff
@@ -240,16 +250,27 @@ public extension JSONAPIEntity {
         var attributes = [String: AnyObject]()
         var relationships = [String: AnyObject]()
         
+        if let linkedEntity  = self as? JSONAPILinkedEntity,
+               entityLinks = linkedEntity.links where entityLinks.count > 0 {
+            data["links"] = linkedEntity.links.serialize()
+        }
+        
         for child in mirror.children {
             if let label = child.label {
                 if let value = child.value as? JSONAPISerializable, let data = value.data(includeRelationships: false, includeAttributes: false) {
                     if includeRelationships {
-                        relationships[label] =  ["data": data]
+                        var relationship = [String: AnyObject](dictionaryLiteral: ("data", data))
```

Comment:
I wanted to directly create the dictionary instead of first specify the type + create. But yes, fixing.